packages/orchestration/README.md

```diff
@@ -10,8 +10,13 @@ This package incorporates generative AI orchestration capabilities into your AI
 4. [Usage](#usage)
    - [Templating](#templating)
    - [Content Filtering](#content-filtering)
-5. [Support, Feedback, Contribution](#support-feedback-contribution)
-6. [License](#license)
+   - [Retrieving Data from the Response](#retrieving-data-from-the-response)
+     - [Finish Reason](#finish-reason)
+     - [Token Usage](#token-usage)
+     - [Using Resource Groups](#using-resource-groups)
+5. [Local Testing](#local-testing)
```

Comment:
I think we should not use a numbered list in ToC at all. I general list should be fine. We run the risk of getting the wrong number ordering.